tests/legacy/test_expansions.py

```
@@ -111,34 +114,35 @@ def test_pack_values_are_strings(self, expansions):
+                    filename = pack["image"].split("/")[-1]
+                    name_without_ext = filename.rsplit(".", 1)[0]
+                    assert name_without_ext == pack["id"], (
+                        f"Pack {pack['id']} image filename '{filename}' doesn't match pack ID"
+                    )
 
     def test_pack_image_file_exists(self, expansions):
         missing = []
         for exp in expansions:
             for pack in exp["packs"]:
-                if pack["id"].startswith("promo"): continue
+                if pack["id"].startswith(("pa-", "pb-")): continue
```

Ruff (0.16.1)

[error] Multiple statements on one line (colon)

(E701)